File: solutions/csharp/darts/1/Darts.cs
using System;

public static class Darts
{

        public static int Score(double x, double y)
        {
            double distance = Math.Sqrt(x * x + y * y);
            if (distance == 0) return 10;
            if (distance <= 1) return 10;
            if (distance >= 1 && distance <= 5) return 5;
            if (distance <= 10 && distance >= 5) return 1;
            return 0;
        }
}

File: solutions/cpp/darts/1/darts.cpp
#include "darts.h"

namespace darts {
    int score(float x, float y)
    {
        int final_score = 0;
        
        if(x == 0 && y == 0)
        {
            final_score = 10;
        }
        else if ((x * x + y * y) <= 1)
        {
            final_score = 10;
        }
        else if ((x * x + y * y) <= 25)
        {
            final_score = 5;
        }
        else if ((x * x + y * y) <= 100)
        {
            final_score = 1;
        }
        return final_score;
    }
    

}  // namespace darts
